docs/modules/user-guide/partials/proc_adding-projects-to-a-devfile.adoc

```diff
@@ -67,16 +71,10 @@ projects:
         origin: "https://github.com/my-org/project1.git"
       checkoutFrom:
         revision: master           <1>
-      tag: 7.2.0
-      commitId: 36fe587
-      branch: master
 ----
-<1> `startPoint` is the general value for `tag`, `commitId`, and `branch`. The `startPoint`, `tag`, `commitId`, and `branch` parameters are mutually exclusive. When more than one is supplied, the following order is used: `startPoint`, `tag`, `commitId`, `branch`.
 ====
 
-`github`:: Same as `git` but for projects hosted on link:https://github.com/[GitHub] only. Use `git` for projects that do not use GitHub-specific features.
-
-`zip`:: Projects with sources in a ZIP archive. Location points to a ZIP file.
+`zip`:: Projects with sources in a ZIP archive. `location` points to a ZIP file.
```

Comment:
"`location` points to a ZIP file" could be a little more explicit, maybe something like this:

```adoc
The `location` property points to the URL of a ZIP archive.
```

Note that I'm not certain that we should refer to `location` as a property, since I don't actually know what the terms are for various things in the Devfile schema. I hope that gets the point across, though. :)